### ISSUE TITLE
Note: If you are using NLLLoss from pytorch make sure to use the log_softmax function from the functional class and not softmax

### ISSUE THREAD

**Issue by rajiv256** [pandeykartikey/Hierarchical-Attention-Network#3]
Hi Pandey, 
Can you tell me the reasoning behind the above statement in your README? 

Thanks.
